plugins/image.md

```diff
@@ -280,6 +280,28 @@ tinymce.init({
 });
 ```
 
+### `image_file_types`
+
+This option configures which image file formats will be uploaded by the `image` plugin.
```

Comment:
does this set which images (based on extension) will be passed the image upload handler (image_upload_handler and/or image_upload_url)?

Reply:
It's more that it prevents them from being recognised as valid images. So any image file types that aren't in the default list (or the overridden list with `image_file_types` setting) will be filtered from the array of files to upload.

https://github.com/tinymce/tinymce/blob/440bd8aeb02d2220547e7d6fa7b93937971c4b0d/modules/tinymce/src/themes/silver/main/ts/ui/dialog/Dropzone.ts#L29

Reply:
maybe:
> can be uploaded by the editor.
?

Reply:
As per discussion on slack, maybe something like this instead?
```
This option configures which image file formats will be accepted by the `image` dialog upload tab.
```